.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+contact_links:
+  - name: Support Request or Question
+    url: https://github.com/jamulussoftware/jamulus/discussions/new
+    about: Please ask questions by opening a new Discussion.
+
+  - name: Idea
```

```suggestion
  - name: Idea/Feature Request 
```


I think this makes it a bit clearer that we don't want feature requests in the issues list since they'll probably need discussion